civis/parallel.py

```diff
@@ -680,25 +726,44 @@ def __init__(self, setup_cmd=_DEFAULT_SETUP_CMD,
                  from_template_id=None,
                  max_submit_retries=0,
                  client=None,
+                 remote_backend='civis',
                  **executor_kwargs):
-        if max_submit_retries < 0:
+        self.setup_cmd = setup_cmd
+        self.from_template_id = from_template_id
+        self.max_submit_retries = max_submit_retries
+        self.client = client
```

Comment:
What's the motivation for changing this from a private attribute to a public one?

Reply:
To make things pickle. There is already public client attached in other places. Thus I just kept it public here so that when I go remove it for pickling I don't have to check both places.

Reply:
Are you saying we were attaching both `client` and `_client` to this object?

Reply:
That makes sense. I think there's no reason this should be public, but we do have it as a public attribute on other objects. Consistency is good.